data/json/items/gun/nail.json

```
@@ -24,5 +24,47 @@
     "reload": 50,
     "valid_mod_locations": [ [ "grip mount", 1 ], [ "rail mount", 1 ], [ "sights mount", 1 ], [ "stock mount", 1 ], [ "underbarrel mount", 1 ] ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "nail": 20 } } ]
+  },
+  {
+    "id": "coilgun",
+    "copy-from": "gun_base",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "coilgun" },
+    "//": "Hard to make, plentiful and cheap ammo, and silent - people will want this thing.",
+    "description": "A homemade gun, using electromagnets to accelerate a ferromagnetic projectile to high velocity.  Powered by UPS.",
+    "weight": "2041 g",
+    "volume": "2750 ml",
+    "longest_side": "46 cm",
+    "price": 75000,
+    "price_postapoc": 5000,
+    "to_hit": -2,
+    "bashing": 10,
+    "material": [ "copper", "steel" ],
+    "ammo": [ "nail" ],
+    "skill": "rifle",
+    "range": 12,
+    "ranged_damage": { "damage_type": "stab", "amount": 1 },
```

```suggestion
    "ranged_damage": { "damage_type": "bullet", "amount": 1 },
```
damaging coilguns are probably closer to high velocity, low momentum attacks like bullets than they are to low velocity, medium momentum attacks like arrows.